src/TodoApp.js

```diff
@@ -0,0 +1,334 @@
+const addBubblingEvent = function(eventName, eventTarget, callback) {
+	document.body.addEventListener(eventName, function(e) {
```

Comment:
```suggestion
	document.body.addEventListener(eventName, (e)  => {
```
이름없는 함수는 이런식으로 arrow function 사용해도 되지 않을까 싶어요~!

Reply:
안녕하세요!
요 며칠간 바빠서 코딩을 못했는데 그 사이에 리뷰를 달아주셨네요!
중간에 확인할게 있어서 에로우를 사용하지 않았는데 아직 수정을 안했습니다!